broadway_api/daos/grading_job_log.py

```diff
@@ -29,6 +29,14 @@ def find_by_id(self, id):
             self._collection.find_one({GradingJobLogDao.ID: ObjectId(id)})
         )
 
+    def find_by_job_id(self, job_id):
+        return list(
+            map(
+                self._from_store,
+                self._collection.find({GradingJobLogDao.GRADING_JOB_ID: job_id}),
```

Comment:
Shouldn't this be `findOne`? I don't think a single grading job id can be associated with multiple jobs (which would remove the need for this function to return a list of results).

Reply:
Ah yes. I'll switch to that